raco/backends/myria/myria.py

```diff
@@ -1524,7 +1524,8 @@ def add_hyper_shuffle():
         if not isinstance(expr, algebra.NaryJoin):
             return expr
         # check if HC shuffle has been placed before
-        shuffled_child = [isinstance(op, algebra.HyperCubeShuffle)
+        shuffled_child = [isinstance(op, algebra.HyperCubeShuffle) or
+                          isinstance(op, algebra.OrderBy)
```

Comment:
wouldn't this only happen if `OrderByBeforeNaryJoin` was somehow applied before `HCShuffleBeforeNaryJoin`?